client/src/components/LinkComponent/LinkComponent.scss

```diff
@@ -1,6 +1,19 @@
 @import '@/variables';
 
+.link-container {
+  display: flex;
+  align-items: center;
+}
+
 .link-component {
   color: $green;
-  text-decoration: underline;
+  cursor: default;
+
+  &.down {
+    color: $application-down-color;
+  }
+}
+
+.link-container .link-component + .state-badge {
+  margin-left: 7px;
```

Comment:
Consider making these classes consistent with BEM styleguiedes

Reply:
Also, is this margin-left necessary here? Shouldn't it be provided by a parent? This would make this component more universal.